my-app/src/App/pages/MarketPage/components/CoinCards/CoinCards.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+import CoinCard from "./CoinCard/CoinCard";
+import styles from "./CoinCards.module.scss";
+import { useCoinContext } from "../../../../App";
+
+const CoinCards: FC = () => {
+  const coinContext = useCoinContext();
+  return (
+    <div className={styles.cards}>
+      {coinContext.length > 0
+        ? coinContext.map((stock) => <CoinCard stock={stock} key={stock.id} />)
+        : ""}
```

Comment:
```suggestion
      {coinContext?.map((stock) => <CoinCard stock={stock} key={stock.id} />)}
```
на пустом массиве можно вызвать метод `map()`, ошибки не будет если что